### ISSUE TITLE
Console output "permission denied"

### ISSUE THREAD

**Issue by hartmutlentz** [Gerungofulus/bvd_agent_based_model#5]
During runtime, there is the following message:
sh: ../BVD_output/testing_time_1300.h5: Permission denied
This happens overtime it writes a file.

Programm is running as normal, though.

**Comment by Gerungofulus** [comment]
I never saw this error before? Did your watchdog run while it was saving? Maybe your watchdog was putting a lock on this file disabling the program to write. This could especially happen if the wile writing process takes some time as your watchdog is listening on file creation. Maybe you can modify your script to wait some time after file creation? Unfortunately I don't have any other explanation for this behavior. Please let me know if it happens more often.

**Comment by Gerungofulus** [comment]
@Yperidis Why do you reopen an issue without writing anything about it?

**Comment by Yperidis** [comment]
Good point! This happens to me as well and a lag in the watchdog daemon could help indeed. I'll let you know of the outcome once I've tried it.

**Comment by hartmutlentz** [comment]
The message might come from the watchdog. And yes, it was running. However, it doesn't seem to have any impact on the program, so give it a low priority.
